src/components/LanguageSelector/LanguageSelector.module.scss

```diff
@@ -0,0 +1,50 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  width: 100%;
+
+  .label {
+    font-size: 14px;
+    font-weight: 500;
+    white-space: nowrap;
+    color: inherit;
+  }
+
+  .select {
+    width: 100%;
+    height: 40px;
+    padding: 8px 12px;
+    border: 2px solid transparent;
+    border-radius: 10px;
```

Comment:
Can we add a shared scss variables sheet and import it here including sizes and border stuff